less/global.less

```diff
@@ -37,10 +38,14 @@ p {
 img {
   max-width: 100%;
   height: auto;
+  border-radius: 10px;
 }
 
 .container {
   max-width: 800px;
   width: 100%;
   margin: 0 auto;
 }
```

Comment:
I would add a media query so that there is some margin or padding on left/right in mobile view. You generally want to avoid text sitting at the edge of the screen.